pipelinewise/cli/pipelinewise.py

```diff
@@ -1233,51 +1228,37 @@ def run_tap(self):
-                    type=target_type,
-                    bin=self.target_bin,
-                    python_bin=self.target_python_bin,
-                    config=cons_target_config,
-                )
+            target_params = TargetParams(
+                target_id=target_id,
+                type=target_type,
+                bin=self.target_bin,
+                python_bin=self.target_python_bin,
+                config=cons_target_config,
+            )
 
-                transform_params = TransformParams(
-                    bin=self.transform_field_bin,
-                    python_bin=self.transform_field_python_bin,
-                    config=tap_transformation,
-                    tap_id=tap_id,
-                    target_id=target_id,
-                )
+            transform_params = TransformParams(
+                bin=self.transform_field_bin,
+                python_bin=self.transform_field_python_bin,
+                config=tap_transformation,
+                tap_id=tap_id,
+                target_id=target_id,
+            )
 
-                # Run fastsync for FULL_TABLE replication method
-                if len(fastsync_stream_ids) > 0:
-                    self.logger.info(
-                        'Table(s) selected to sync by fastsync: %s', fastsync_stream_ids
-                    )
-                    self.tap_run_log_file = os.path.join(
-                        log_dir, f'{target_id}-{tap_id}-{current_time}.fastsync.log'
-                    )
-                    tap_params = TapParams(
-                        tap_id=tap_id,
-                        type=tap_type,
-                        bin=self.tap_bin,
-                        python_bin=self.tap_python_bin,
-                        config=tap_config,
-                        properties=tap_properties_fastsync,
-                        state=tap_state,
-                    )
+            # Run fastsync for FULL_TABLE replication method
+            if len(fastsync_stream_ids) > 0:
+                self.logger.info(
+                    'Table(s) selected to sync by fastsync/partialsync: %s', fastsync_stream_ids
+                )
+                self.sync_tables()
 
-                    self.run_tap_fastsync(
-                        tap=tap_params, target=target_params, transform=transform_params
-                    )
-                else:
-                    self.logger.info(
-                        'No table available that needs to be sync by fastsync'
-                    )
+            else:
+                self.logger.info(
+                    'No table available that needs to be sync by fastsync'
+                )
 
-                # Run singer tap for INCREMENTAL and LOG_BASED replication methods
-                if len(singer_stream_ids) > 0:
+            # Run singer tap for INCREMENTAL and LOG_BASED replication methods
+            if len(singer_stream_ids) > 0:
+                with pidfile.PIDFile(self.tap['files']['pidfile']):
```

Comment:
what's the reason for changing the usage of PID file to open one for fastsync/partial sync and another one for singer separately?

Reply:
it was for testing and missed.

- [ ] fix it